docker/sbatch_example.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#SBATCH --account=cvgl
+#SBATCH --partition=svl --qos=normal
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=8
+#SBATCH --mem=30G
+#SBATCH --gres=gpu:2080ti:1
+
+IMAGE_PATH="/cvgl2/u/cgokmen/omnigibson.sqsh"
+GPU_ID=$(nvidia-smi -L | grep -oP '(?<=GPU-)[a-fA-F0-9\-]+' | head -n 1)
+ISAAC_CACHE_PATH="/scr-ssd/${SLURM_JOB_USER}/isaac_cache_${GPU_ID}"
+
+# Define env kwargs to pass
+declare -A ENVS=(
+    [NVIDIA_DRIVER_CAPABILITIES]=all
+    [NVIDIA_VISIBLE_DEVICES]=0
+    [DISPLAY]=""
+    [OMNIGIBSON_HEADLESS]=1
+)
+for env_var in "${!ENVS[@]}"; do
+    # Add to env kwargs we'll pass to enroot command later
+    ENV_KWARGS="${ENV_KWARGS} --env ${env_var}=${ENVS[${env_var}]}"
+done
+
+# Define mounts to create (maps local directory to container directory)
+declare -A MOUNTS=(
+    [/scr-ssd/og-data-0-2-1]=/data
+    [${ISAAC_CACHE_PATH}/isaac-sim/kit/cache/Kit]=/isaac-sim/kit/cache/Kit
+    [${ISAAC_CACHE_PATH}/isaac-sim/cache/ov]=/root/.cache/ov
+    [${ISAAC_CACHE_PATH}/isaac-sim/cache/pip]=/root/.cache/pip
+    [${ISAAC_CACHE_PATH}/isaac-sim/cache/glcache]=/root/.cache/nvidia/GLCache
+    [${ISAAC_CACHE_PATH}/isaac-sim/cache/computecache]=/root/.nv/ComputeCache
+    [${ISAAC_CACHE_PATH}/isaac-sim/logs]=/root/.nvidia-omniverse/logs
+    [${ISAAC_CACHE_PATH}/isaac-sim/config]=/root/.nvidia-omniverse/config
+    [${ISAAC_CACHE_PATH}/isaac-sim/data]=/root/.local/share/ov/data
+    [${ISAAC_CACHE_PATH}/isaac-sim/documents]=/root/Documents
+    # Feel free to include lines like the below to mount a workspace or a custom OG version
+    # [/cvgl2/u/cgokmen/OmniGibson]=/omnigibson-src
+    # [/cvgl2/u/cgokmen/my-project]=/my-project
+)
+
+MOUNT_KWARGS=""
+for mount in "${!MOUNTS[@]}"; do
+    # Verify mount path in local directory exists, otherwise, create it
+    if [ ! -e "$mount" ]; then
+        mkdir -p ${mount}
+    fi
+    # Add to mount kwargs we'll pass to enroot command later
+    MOUNT_KWARGS="${MOUNT_KWARGS} --mount ${mount}:${MOUNTS[${mount}]}"
+done
+
+# Create the image if it doesn't already exist
+CONTAINER_NAME=omnigibson_${GPU_ID}
+enroot create --force --name ${CONTAINER_NAME} ${IMAGE_PATH}
```

Comment:
this is awesome!! so just to be clear -- this automatically creates the enroot image if needed? so there's only a one-time slowdown when it is first created? (also, so there's no need for a .sqsh file then?)